src/pages/Contact/ContactPage.jsx

```diff
@@ -32,30 +64,64 @@ const ContactPage = () => {
           <S.Title>Contact</S.Title>
           <S.SubTitle>연락 정보</S.SubTitle>
         </S.TitleContainer>
-        <S.CardContainer>
-          <S.ContactCardContainer onClick={() => clickCopy('010-4598-2934')}>
+        <S.CardContainer {...animation}>
+          <S.ContactCardContainer onClick={() => clickCopy(classInfo.phoneNum)}>
             <S.PhoneIcon />
-            <S.CopySubTitle>37th 회장 전유나</S.CopySubTitle>
-            <S.Content>010-4598-2934</S.Content>
+            <S.CopySubTitle>{classInfo.phoneNumInfo}</S.CopySubTitle>
+            <S.Content>{classInfo.phoneNum}</S.Content>
+          </S.ContactCardContainer>
+          <S.ContactCardContainer onClick={() => clickCopy(classInfo.email)}>
+            <S.EmailIcon />
+            <S.CopySubTitle>E-mail</S.CopySubTitle>
+            <S.Content>{classInfo.email}</S.Content>
           </S.ContactCardContainer>
           <S.ContactCardContainer
-            onClick={() => clickCopy('ewha_epris@naver.com')}
+            onClick={() => clickCopy(classInfo.instaLink)}
+          >
+            <S.InstagramIcon />
+            <S.CopySubTitle>Instagram</S.CopySubTitle>
+            <S.Content>{classInfo.instaLink}</S.Content>
```

Comment:
```suggestion
            <S.Content>@epris_ewha</S.Content>
```
주소 값 보여주는 거보다 아이디 표시하는 게 낫지 않나요!?